net-libs/libwebsockets/libwebsockets-2.1.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="canonical libwebsockets.org websocket library"
+HOMEPAGE="https://libwebsockets.org/"
+SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+http2 +ssl cgi access-log server-status client http-proxy ipv6 libev libressl libuv static-libs"
```

Comment:
I apologize if we had this conversation before (I feel as though we did), but why aren't we adding support for `sqlite3` and others as well? https://github.com/warmcat/libwebsockets/compare/v2.0.2...v2.1.1#diff-af3b638bc2a3e6c650974192a53c7291R108